packages/tackle-box/lib/components/Stack/Stack.tsx

```diff
@@ -12,26 +12,62 @@ const styles = css.create({
   gap: (gap: number) => ({
     gap,
   }),
+  align: (align: string) => ({
+    alignItems: align,
+  }),
+  justify: (justify: string) => ({
+    justifyContent: justify,
+  }),
 });
 
 type StackProps = BoxProps & {
   gap?: number;
+  spacing?: number;
+  align?: string;
+  justify?: string;
 };
 
-export function HStack({ gap = 0, children, ...rest }: StackProps) {
+export function HStack({
+  gap = 0,
+  spacing,
+  align = "stretch",
+  justify = "flex-start",
+  children,
+  ...rest
+}: StackProps) {
   return (
     <Box
-      style={[styles.horizontal, styles.gap(applyBaseSpacing(gap))]}
+      style={[
+        styles.horizontal,
+        styles.gap(applyBaseSpacing(spacing ?? gap)),
+        styles.align(align),
+        styles.justify(justify),
+      ]}
       {...rest}
     >
       {children}
     </Box>
   );
 }
 
-export function VStack({ gap = 0, children, ...rest }: StackProps) {
+export function VStack({
+  gap = 0,
+  spacing,
+  align = "stretch",
+  justify = "flex-start",
```

Comment:
issue?: Since threre's align-items, align-self, and align-content properties (similar thing for justify), would it make more sense to use alignItems and justifyContent?

Reply:
I just copied Chakra so I also don't feel strongly to that and agree it is easier to read if its the full property name